README.md

```diff
@@ -26,6 +27,17 @@ unzip phoenicis-dist.zip
 ./phoenicis-dist/phoenicis.sh
 ```
 
+### OSX
+Build
+```
+mvn clean package install
```

Comment:
Why do `install` here if it's not used to run Phoenicis (see below)?

Reply:
Because the plugin is fetching dependencies from the maven repository

Reply:
I don't quite understand. The required dependencies should be downloaded during build anyways.

Reply:
I mean phoenicis-* dependencies

Reply:
I thought the purpose was to get a fat .jar with all dependencies?

Reply:
a .dmg to be more precise. However the dependencies are not grabbed from the current source tree but from your local maven repository

Reply:
I see. So we cannot include the .dmg build in the normal build process because phoenicis must be installed first? I somehow assumed that the plugin could pick up the project internal dependencies without having to install them.

Reply:
We cannot, not because of mvn install (I think that most of the CI platform will not have problem with that). However, OSX tools are needed to create the .dmg, so we must have a Apple machine so far to do it

Reply:
You can select `os: osx` in Travis. Don't you think that is sufficient?

Reply:
I don't think so because I also need to provide my personal certificate to sign the package...

Reply:
I know, this is really a pain

Reply:
Oh.